src/modules/utils.js

```diff
@@ -29,3 +29,15 @@ export const useDidUpdateEffect = (fn, inputs) => {
 
 export const formatDimensionId = (dimensionId, programStageId) =>
     programStageId ? `${programStageId}.${dimensionId}` : dimensionId
+
+export const extractDimensionParts = (input) => {
```

Comment:
Could you add some unit tests for the function? That would also help to document what it does.

Also thinking that the function name should indicate that it is operating on the dimensionId, not the dimension object (which contains name and other properties).  Perhaps getDimensionIdParts, or extractDimensionIdParts

Reply:
Renamed + added unit tests 👍